src/abstractdataframe/io.jl

```diff
@@ -290,13 +295,13 @@ function _show(io::IO, ::MIME"text/latex", df::AbstractDataFrame; rowid=nothing)
     write(io, "\\end{tabular}\n")
 end
 
-function Base.show(io::IO, mime::MIME"text/latex", dfr::DataFrameRow)
+function Base.show(io::IO, mime::MIME"text/latex", dfr::DataFrameRow; eltypes::Bool=true)
     r, c = parentindices(dfr)
-    _show(io, mime, view(parent(dfr), [r], c), rowid=r)
+    _show(io, mime, view(parent(dfr), [r], c), rowid=r, eltypes=eltypes)
 end
 
-Base.show(io::IO, mime::MIME"text/latex", dfrs::DataFrameRows) = _show(io, mime, parent(dfrs))
-Base.show(io::IO, mime::MIME"text/latex", dfcs::DataFrameColumns) = _show(io, mime, parent(dfcs))
+Base.show(io::IO, mime::MIME"text/latex", dfrs::DataFrameRows; eltypes::Bool=true) = _show(io, mime, parent(dfrs), eltypes=eltypes)
```

Comment:
can you split the lines here so that they are not so long (also in other places in this PR)? Probably splitting at `=` is OK

Reply:
split in de7e4e9